.github/workflows/build.yml

```
@@ -365,7 +365,7 @@ jobs:
           if_true: 'stable'
           if_false: 'latest'
 
-      - name: Update GitHub release
+      - name: Update GitHub release (stable / latest tag)
```

```suggestion
      - name: Update GitHub release (latest tag)
        if: ${{ env.IS_RELEASE == 'false'}}
```
What do you think about this?

---

Yep, that should work